cdc/owner/ddl_manager.go

```diff
@@ -155,57 +173,74 @@ func newDDLManager(
 		zap.Uint64("checkpointTs", checkpointTs),
 		zap.Bool("bdrMode", bdrMode))
 
+	bootstrap := bootstrapNotStarted
+	if !shouldSendAllBootstrapAtStart {
+		bootstrap = bootstrapFinished
+	}
+
 	return &ddlManager{
-		changfeedID:                   changefeedID,
-		ddlSink:                       ddlSink,
-		filter:                        filter,
-		ddlPuller:                     ddlPuller,
-		schema:                        schema,
-		redoDDLManager:                redoManager,
-		redoMetaManager:               redoMetaManager,
-		startTs:                       startTs,
-		checkpointTs:                  checkpointTs,
-		ddlResolvedTs:                 startTs,
-		BDRMode:                       bdrMode,
-		pendingDDLs:                   make(map[model.TableName][]*model.DDLEvent),
-		shouldSendAllBootstrapAtStart: shouldSendAllBootstrapAtStart,
+		changfeedID:     changefeedID,
+		ddlSink:         ddlSink,
+		filter:          filter,
+		ddlPuller:       ddlPuller,
+		schema:          schema,
+		redoDDLManager:  redoManager,
+		redoMetaManager: redoMetaManager,
+		startTs:         startTs,
+		checkpointTs:    checkpointTs,
+		ddlResolvedTs:   startTs,
+		BDRMode:         bdrMode,
+		pendingDDLs:     make(map[model.TableName][]*model.DDLEvent),
+		bootstrap:       bootstrap,
```

Comment:
Bootstrap is verb here, not match with bool type.

Reply:
it's not a bool, it's a enum

Reply:
OK, but "enum" is also a noun and "bootstrap" is also a verb. It does seem a bit odd.